core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -107,7 +110,7 @@ class DagrCoreMain(
   private var reportPath  : Option[Path] = None
 
   // Initialize the configuration as early as possible
-  Configuration.initialize(this.config)
+  dagr.sopt.util.Configuration.initialize(this.config)
```

Comment:
Something about this feels wrong.  Firstly, I think perhaps the main `Configuration` trait should be pushed down to commons.  Secondly I don't love that `dagr` is initializating `sopt`'s configuration.  Perhaps the pattern should be that any toolkit/app that uses needs config extends the `Configuration` trait from `commons` to provide it with access to the appropriate config instance?